helper-cli/src/main/kotlin/commands/curationshelper/CreatePackageCurationCommand.kt

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.helper.commands.curationshelper
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.UsageError
+import com.github.ajalt.clikt.parameters.options.convert
+import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.enum
+import com.github.ajalt.clikt.parameters.types.file
+
+import java.io.IOException
+
+import org.ossreviewtoolkit.helper.common.getSplitCurationFile
+import org.ossreviewtoolkit.model.FileFormat
+import org.ossreviewtoolkit.model.Identifier
+import org.ossreviewtoolkit.model.PackageCuration
+import org.ossreviewtoolkit.model.PackageCurationData
+import org.ossreviewtoolkit.model.writeValue
+import org.ossreviewtoolkit.utils.expandTilde
+
+internal class CreatePackageCurationCommand : CliktCommand(
+    help = "Create a single curation file for a package id complying to the directory structure created by the" +
+            "`split-curations` sub-command."
+) {
+    private val packageId by option(
+        "--package-id",
+        help = "The target package id for which a curation template should be generated."
+    ).convert { Identifier(it) }
+        .required()
+
+    private val outputDir by option(
+        "--package-curations-dir",
+        help = "The parent directory where the curation template should be created."
+    ).convert { it.expandTilde() }
+        .file(mustExist = true, canBeFile = false, canBeDir = true, mustBeWritable = true, mustBeReadable = false)
+        .convert { it.absoluteFile.normalize() }
+        .required()
+
+    private val fileFormat by option(
```

Comment:
Is this really needed? "helper-cli" so far IIRC only outputs YAML and I prefer to keep that unless there is a specific requirement.

Reply:
It's not required for our use case. I'll remove it if the intention of the `helper-cli` is not intended to support all of ORT's data file formats.

Reply:
What the intention of the `helper-cli` is is not written in stone, it's also not supposed to be dictated by a single person but ideally a result of collective work changing iteratively.

Anyhow, I'd propose to drop it for consistency (at least) still and as I believe it falls under YAGNI.